ldap/servers/plugins/uiduniq/uid.c

```diff
@@ -820,16 +820,16 @@ searchAllSubtrees(Slapi_DN **subtrees, Slapi_DN **exclude_subtrees, const char *
 
     /*
    * For each DN in the managed list, do uniqueness checking if
-   * the target DN is a subnode in the tree.
+   * the destination (target) DN is a subnode in the tree.
```

Comment:
I agree it is destination but the word 'target' is IMHO confusing.

Reply:
searchAllSubtrees is also used for modify and add, and it uses targetSDN parameter. So I left it here for people who will read this function after preop_modify or preop_add, so they can easily understand (and not being confused by "destination")